### ISSUE TITLE
Migrate from tailwind to ornament + girouette

### ISSUE THREAD

**Issue by oxalorg** [lambdaisland/gaiwan_co#24]
There's too much tailwind stuff in there. Maybe start migrating iteratively.

**Comment by plexus** [comment]
Let's not create busywork for ourselves, the site is working, and if it's about adding small stuff or doing tweaks what is there is fine. If we add or rework significant portions we can start doing that with Ornament.

I would boyscout-wise start splitting some of the bigger components into smaller functions each time we have reason to touch the site. Converting these to Ornament at some later point should be fairly mechanical.

**Comment by oxalorg** [comment]
> each time we have reason to touch the site

Yup I agree! This is what I had in mind when I said "iteratively" 🙈.
